### PR TITLE
Imp: add response

### PR DESCRIPTION
**Pull request by fangyincheng**

### CODE REVIEW

File: response.go [hunk 1]
```diff
@@ -135,19 +141,21 @@ func unpackResponseBody(buf []byte, rspObj interface{}) error {
...
 		rsp, err := decoder.Decode()
 		if err != nil {
 			return perrors.WithStack(err)
 		}
-		return perrors.WithStack(ReflectResponse(rsp, rspObj))
+		return perrors.WithStack(ReflectResponse(rsp, response.RspObj))
 
 	case RESPONSE_NULL_VALUE, RESPONSE_NULL_VALUE_WITH_ATTACHMENTS:
-		return perrors.New("Received null")
+		return nil
```
Comment (AlexStocks) [inline response.go]: Why use nil instead of a error?

Comment (fangyincheng) [reply response.go]: `RESPONSE_NULL_VALUE` and `RESPONSE_NULL_VALUE_WITH_ATTACHMENTS` are normal.
They should not be an error, and that is similar to `RESPONSE_WITH_EXCEPTION`.

**Comment by wongoo** [comment]
pls follow the PR template to give more details next time, so that we can make sure what it's changing, thanks!

**Comment by AlexStocks** [comment]
> pls follow the PR template to give more details next time, so that we can make sure what it's changing, thanks!

The java exception can not be handled as network error.

File: response.go (hunk 1 above)
Comment (AlexStocks) [reply response.go]: got it.

**Comment by fangyincheng** [comment]
> pls follow the PR template to give more details next time, so that we can make sure what it's changing, thanks!

sorry, I will do it next time.

**Pull request merged**